static/js/admin.js

```diff
@@ -959,13 +959,15 @@ function _setup_page() {
         var aliases_info = $("#realm_aliases_modal").find(".aliases_info");
         var data = {
             domain: JSON.stringify($("#new_alias").val()),
+            allow_subdomains: JSON.stringify($("#allow_subdomains").prop("checked")),
         };
 
         channel.post({
             url: "/json/realm/domains",
             data: data,
             success: function () {
                 $("#new_alias").val("");
+                $("#allow_subdomains").prop("checked", false);
```

Comment:
Ah, that makes sense you can probably make this code more clear by cleaning up the HTML classes.  The container for the new alias could be "#add-alias" and then the widgets could be ".alias-name" and ".alias-allow-subdomains" and then you could say something like

~~~ js
var add_row = $("add-alias");
add_row.find(".alias-name").val("");
add_row.find(".alias-allow-subdomains").prop("checked", false);